debugger.md

```diff
@@ -15,8 +15,8 @@ Please place your feedback [here](https://github.com/OmniSharp/omnisharp-vscode/
 * X64 only
 * Basic Debugging Setup successfully validated on Ubuntu Linux and Mac
 * Currently NOT working on Windows
-* In case you get this error on F5: *"No task runner configured -  Tasks.json could not be found"*, see the 'once for each project' section below. Currently manual steps are required.
-* If you don’t have mono installed you won’t get IntelliSense. See optional installation section below.
+* In case you get this error on F5: *"No task runner configured -  Tasks.json could not be found"*, see the 'once for each project' section below. 
+* If you don’t have mono installed you won’t get IntelliSense. 
```

Comment:
you have a 'word quote'. Might want to convert these to normal (I don't know how well they will render on other OSs).